code/src/GridAPI/Rows.ts

```diff
@@ -15,20 +15,44 @@ namespace GridAPI.Rows {
         gridID: string,
         rowNumber: number,
         className: string
-    ): void {
+    ): string {
         PerformanceAPI.SetMark('Rows.AddClass');
+        let returnMessage = {
+            isSuccess: true,
+            message: 'Success',
+            code: OSFramework.Enum.ErrorCodes.GRID_SUCCESS
+        };
 
-        const grid = GridManager.GetGridById(gridID);
-
-        if (grid !== undefined) {
-            grid.features.rows.addClass(rowNumber, className, true);
+        if (!OSFramework.Helper.IsGridReady(gridID)) {
+            returnMessage = {
+                isSuccess: false,
+                message: 'Grid not found',
```

Comment:
This text should not be hardcoded here. We should have an Enum ErrorMessages. Repeated with https://github.com/OutSystems/outsystems-datagrid-reactive/pull/188

Reply:
done